CODINGSTYLE.md

```diff
@@ -140,6 +140,14 @@ Example:
         return false;
     }
 ```
+Prefer to use std algorithms as much as possible.
+Get to know std algorithms.
+std algorithms are better than hand crafted loops.
+std algorithms are thoughful, and tested, and do as expected.
```

Comment:
Is it a misspell of `thoughtful`? If so, what does it mean? I think a better way to say this is:
```
std algorithms are thoroughly tested, well documented, and do exactly as expected.
```